File: Level_2/6_Remainder/Remainder.java
import java.util.Scanner;

class Remainder{
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);

        int left, center, right;

        left = scan.nextInt();
        center = scan.nextInt();
        right = scan.nextInt();

        System.out.println((left + center) % right);
        System.out.println((left % right + center % right) % right);
        System.out.println((left * center) % right);
        System.out.println((left % right * center % right) % right);

        scan.close();
    }
}

File: Level_2/6_Remainder/Remainder.cpp
#include <iostream>
using namespace std;

int main(){
    int left, center, right;

    cin>>left>>center>>right;

    cout<<(left + center) % right<<endl;
    cout<<(left % right + center % right) % right<<endl;
    cout<<(left * center) % right<<endl;
    cout<<(left % right * center % right) % right<<endl;
    return 0;
}